Gio-2.0.gir

```diff
@@ -31602,6 +31649,53 @@ move operation isn't available).</doc>
           </parameter>
         </parameters>
       </method>
+      <method name="move_async" c:identifier="g_file_move_async" introspectable="0">
```

Comment:
This needs some manual bindings

Reply:
And needs https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/319